utxo/utxo.go

```diff
@@ -1349,7 +1349,7 @@ func getGasLimitFromTx(tx *pb.Transaction) (int64, error) {
 		return gasLimit, nil
 	}
 	// FIXME: 没有小费的tx如何得到gas limit?
-	return 0, ErrGasNotEnough
+	return 0, nil
```

Comment:
In the past, the function of getGasLimitFromTx was used to get gas limit for contract and It was nothing with the ordinary transfer transaction.
But, now we have to consider the return value of the function of getGasLimitFromTx has to be campatiable with all kinds of transactions because of reserved contracts, we change the return values of 0, nil instead of 0, ErrGasNotEnough.